src/main.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+import { NFC } from 'nfc-pcsc';
+import EthTx from 'ethereumjs-tx';
+import LuksoNfcChip from './reader/LuksoNfcChip';
+import Tlv from './tlv/Tlv';
+
+const nfc = new NFC();
+
+nfc.on('reader', async reader => {
+  console.log(`${reader.reader.name}  device attached`);
+  reader.aid = '654E6F7465734170706C6574';
+  reader.on('card', async card => {
+    console.log();
+
+    // Get Chip Object (Automatic read public key)
+    let chip = await LuksoNfcChip.build(reader);
```

Comment:
We pass the reader object. Wouldn’t it be better to pass a structured object, so that it can adopt to any nfc reader library?

Reply:
I don't think so. We need to send APDU comand via the reader, and this function is defined in reader object.